docs/usage/secrets_rotation.md

```diff
@@ -47,6 +47,8 @@ The substituted Kubeconfig are provided after the initialized reconciliation was
 which were created with Gardener version `<= 0.28.0` used to have a Kubeconfig based on a client certificate instead of a static token.
 These client certificates are not revocable and thus a full credential rotation is not supported.
 
+You can check the `.status.credentials.rotation.kubeconfig` field in the `Shoot` to see when the rotation was last initiated or last completed.
```

Comment:
Can you add checks for the status fields to the existing testmachinery test? https://github.com/gardener/gardener/blob/19d3d30b4097c131c30fb6632158dcc4ae374eb4/test/testmachinery/shoots/operations/operations.go#L120